sdk/include/asm/asm.inc

```diff
@@ -189,8 +189,23 @@ data32 MACRO opcode:VARARG
     opcode
 ENDM
 
-UNIMPLEMENTED MACRO name
+UNIMPLEMENTED2 MACRO file, line, func
+    jmp UNIMPLEMENTED2_IMPL
+UNIMPLEMENTED_MSG: .ascii "WARNING:  %s at %s:%d is UNIMPLEMENTED!", 10, 0
+UNIMPLEMENTED_FUNC: .ascii "&func&", 0
+UNIMPLEMENTED_FILE: .ascii file, 0
+EXTERN DbgPrint:PROC
+UNIMPLEMENTED2_IMPL:
+    sub rsp, 28h
+    mov r9, line
+    lea r8, UNIMPLEMENTED_FILE
+    lea rdx, UNIMPLEMENTED_FUNC
+    lea rcx, UNIMPLEMENTED_MSG
+    call DbgPrint
+    add rsp, 28h
+    xor eax, eax
```

Comment:
The GAS version does not do this, but it seems safer to not leave DbgPrint's result in there.

Reply:
Could then be of interest to port it back to GAS 🤔

Reply:
I don't have a gcc64 build setup, and I don't really like editing code without properly testing it.